mlflow/utils/search_utils.py

```diff
@@ -105,15 +105,23 @@ def get_sql_filter_ops(cls, column, operator, dialect):
         if dialect == MSSQL:
             column = column.collate("Japanese_Bushu_Kakusu_100_CS_AS_KS_WS")
 
+        # Use non-binary ahead of binary comparison for runtime performance
+        col = f"{column.class_.__tablename__}.{column.key}"
         # Use non-binary ahead of binary comparison for runtime performance
         def case_sensitive_mysql_eq(value):
-            return sa.and_(column.__eq__(value), column.op("= BINARY", is_comparison=True)(value))
+            return sa.text(f"{col} = :value AND BINARY {col} = :value").bindparams(
```

Comment:
So this is because `column.op("= BINARY")` does not work ?

Reply:
yes